hokusai/development.yml

```diff
@@ -0,0 +1,36 @@
+---
+version: '2'
+services:
+  aprb:
+    environment:
+    - MIX_ENV=dev
+    - DB_USER=postgres
+    - DB_NAME=postgres
+    - DB_HOST=postgres
+    - DB_PASSWORD=postgres
+    - RABBITMQ_HOST=rabbitmq
+    - RABBITMQ_USER=guest
+    - RABBITMQ_PASSWORD=guest
+    - RABBITMQ_PORT=5672
+    - GRAVITY_API_URL=https://api.artsy.net/api
+    - GRAVITY_API_TOKEN=foo
+    - SLACK_API_TOKEN=bar
+    - SLACK_SLASH_COMMAND_TOKEN=baz
+    extends:
+      file: common.yml
+      service: aprb
+    ports:
+    - 4000:4000
+  postgres:
```

Comment:
do we need to make sure to persist the data somewhere for development, otherwise everytime we start stop the machine we'd lose the database, right? I remember before when I did this I had to make sure we persist the data.